### ISSUE TITLE
Editor: the new 'Back' button is partially obscured with 'Show button text labels' preference on

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#81680]
Originally reported at https://core.trac.wordpress.org/ticket/65882 by @afercia 

The 'Show button text labels' preference in the editor is often overlooked. It is rarely tested manually and rarely new designs take it into account. In the history of the editor, it often broke.

I would like to see more awareness about this feature and more importantly more E2E tests for it as it often breaks when new feature are added.

For the new 'Back' button in the editor, the button appears to have a fixed with. So that when 'Show button text labels' is on, the width doesn;t enlarge and the following controls stay there, partially obscuring it.

See screenshot.

To reproduce:
- Edit a post.
- Go to Options > Preferences > Accessibility > Show button text labels.
- Enable Show button text labels.
- Close the preferences modal dialog.
- Observe the Back button is partially obscured,

<img width="422" height="273" alt="Image" src="https://github.com/user-attachments/assets/ce335ab6-89ab-4810-ad62-9e39186dd80f" />

Same problem in the Site editor.

<img width="433" height="141" alt="Image" src="https://github.com/user-attachments/assets/91d00529-c6be-40be-b9ac-bce05d9a6222" />

**Comment by afercia** [comment]
> appears to have a fixed with.

A note that, actually, it's the `grid-template` property on the header container that makes the Back button not expand when it shows text:

`grid-template: auto / 32px minmax(min-content, 2fr) 2.5fr minmax(min-content, 2fr) 64px;`

the `32px` value there should change to `auto` when the editor layout has the class `show-icon-labels`.

A note that I would recommend to double check the `64px` value as well. I don't know for what it is used but it deserves some investigation.

**Comment by afercia** [comment]
Thank you @t-hamano .
While I understand that https://core.trac.wordpress.org/ticket/65882 needed to be reported upstream, I'm not sure why this issue is not milestoned.

This is a _new, known, bug_ introduced in 7.1. It should be fixed before release.
Can you please share why this issue is not milestoned for 7.1? Thank you.

**Comment by t-hamano** [comment]
> Can you please share why this issue is not milestoned for 7.1? 

It was my oversight that I didn't make the milestone for 7.1. We should fix this in the 7.1 release, but we have very limited resources and time before the commit freeze. I will do my best, but it might have to be pushed to a minor release. Especially since the CSS for the header is very complex, I hope that fixing the back button issue hasn't caused any other problems.
